src/foam/dao/ManyToManyRelationshipDAO.js

```diff
@@ -18,51 +18,38 @@
 foam.CLASS({
   package: 'foam.dao',
   name: 'ManyToManyRelationshipDAO',
-  extends: 'foam.dao.RelationshipDAO',
+  extends: 'foam.dao.ProxyDAO',
 
   implements: [ 'foam.mlang.Expressions' ],
 
   documentation: 'Adapts a DAO based on a *:* Relationship.',
 
   properties: [
```

Comment:
Can we please document these? Every time I go to debug relationships I get confused about "is this 'target' always 'target', or is it 'target, unless some state indicates that this relationship is inverted'?", and "this 'junction*' property, does it refer to the whole junction DAO? A filtered DAO with only `this.obj`'s junctions?", likewise for source/target DAOs, etc..